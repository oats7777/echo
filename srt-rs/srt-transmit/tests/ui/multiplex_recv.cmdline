["srt://:2224?multiplex" ,  "udp://127.0.0.1:4000"]