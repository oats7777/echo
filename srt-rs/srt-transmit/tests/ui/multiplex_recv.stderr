Invalid settings detected: multiplex is not a valid option for input urls

See srt-transmit --help for more info
